HomeTask4/src/Task4Array.java

```diff
@@ -0,0 +1,73 @@
+import java.util.Scanner;
+
+public class Task4Array {
+    public static void main(String[] args) {
+        int filter = 0;
+
+        System.out.print("Please input three words separated by the comma: ");
+        Scanner scanner = new Scanner(System.in);
+        String words = scanner.next();
+        String[] wordsArray = words.split(",");
+
+        do {
+            System.out.println("Choose type of filter: ");
+            System.out.println("1 - maximum length");
+            System.out.println("2 - starts with");
+            System.out.println("3 - ends with");
+            System.out.println("4 - contains");
+            System.out.println("0 - EXIT");
+            filter = scanner.nextInt();
+
+
+            switch (filter) {
+                case 1:
+                    System.out.print("Input maximum word length: ");
+                    int maxLength = scanner.nextInt();
+                    for (int i = 0; i < wordsArray.length; i++) {
```

Comment:
Please replace such cases with foreach (for (variable : array)) (here and below)